oci/spec_opts.go

```diff
@@ -347,8 +366,9 @@ func WithImageConfigArgs(image Image, args []string) SpecOpts {
 			return err
 		}
 		var (
-			ociimage v1.Image
-			config   v1.ImageConfig
+			ociimage         v1.Image
+			config           v1.ImageConfig
+			ociImageExtended imageExtended
```

Comment:
Perhaps change this to just the `ArgsEscaped` that was detected (see other comment below)

```suggestion
			argsEscaped bool
```

Reply:
Sure np